src/components/BusinessTable/BusinessTable.jsx

```diff
@@ -62,8 +71,46 @@ const BusinessTable = () => {
     'Created Date',
     'City',
   ];
-
   const tableHeaders = TABLE_HEADERS.map(tableHeader => <th key={tableHeader}>{tableHeader}</th>);
+  const [checkedItems, setCheckedItems] = useState(new Array(TABLE_HEADERS.length).fill(false));
```

Comment:
Have the id of the checked business instead of having an array of false for scalability reasons. Should be an empty array and then add a business id when a business is checked.